compiler/lexer.nim

```diff
@@ -139,6 +139,9 @@ type
                               # needs so much look-ahead
     currLineIndent*: int
     strongSpaces*, allowTabs*: bool
+    inAccent*: int            # if this is 1, we're inside backticks. Used to
```

Comment:
Why is this an int and not a `bool`?

Reply:
Uhm, because when I hacked it together originally a few weeks ago, I wanted to toggle it with `xor` to avoid 
`L.inAccent = if L.inAccent: false else: true`. 
I agree that it should be a `bool` though. Is there a builtin to toggle a bool or is this the most succinct?

Reply:
You could write `L.inAccent = not L.inAccent`

Reply:
Fixed.

Reply:
@Stromberg90 A brain is a funny thing. Me writing `not L.inAccent` a few lines below, but not thinking about it to flip on assignment. Sigh.